### DISCUSSION TITLE
How to re focus on text field ?

### DISCUSSION THREAD

**Discussion by SoufAsd** [vuetifyjs/vuetify#20540] [Q&A]
```
const handleBlur = () => {
  if (myTextField.value) {
    myTextField.value.focus();
  }
};
```

it seems like focus() doesnt work to re focus on text field once blur is triggered 
is there a way of doing it ?

**Comment by angrymarker** [comment]
this works for me
https://play.vuetifyjs.com/#eNpVkDGPwjAMhf+KlalI0C43nUJ1N7AxslGGElwUyU2ixKmQUP87aVIkWJLnF/vzU85PEbxq/p2rp4jiV0jG0VHP2HYGQE673rksc6Gs4V4b9KuVTcYH7waNdAPqr0j7ThyXuxPwd6XoU+1xsCqGTrSy+Rx4k5P5jV6cslk2H4lkUF47hoAcS6whGsXaGlhXVDih4Q08CzlRgyWsyd7Xl+InwEmPaCNX1Qb27bsfIHfV6VMOhOMiC3adm7fwk+W8JCtpUg4xb00kysflBUxHcTo=
